### PR TITLE
:lipstick: Add start/end-tunnels and placeholder + functionality

### PR DESCRIPTION
**Pull request by pierman1**

### CODE REVIEW

File: app/components/parts/author/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import html from 'choo/html';
 import styles from '../parts.css';
 
+console.log(styles);
```
Comment (rijkvanzanten) [inline app/components/parts/author/index.js]: @pierman1 dangit boy

File: app/components/parts/author/index.js [hunk 2]
```diff
@@ -24,24 +26,39 @@ export const modalSettings = {
...
-    <defs>
-      <path id="machineauthor-a" d="M8.36440678,4.99582647 C8.36440678,2.2367077 10.611347,0 13.3612887,0 L84.223457,0 C86.9831587,0 89.220339,2.24473649 89.220339,4.99582647 L89.220339,54.6623377 L8.36440678,54.6623377 L8.36440678,4.99582647 Z"/>
-    </defs>
-    <g fill="none" fill-rule="evenodd">
-      <g transform="translate(133 76)">
-        <rect width="14.359" height="43.256" x="7.179" y="1.291" fill="#9B9B9B"/>
-        <rect width="27.282" height="5.52" x=".718" y="40.48" fill="#4A4A4A"/>
-        <rect width="7.179" height="43.256" x="7.179" y="1.291" fill="#4A4A4A"/>
-        <rect width="27.282" height="5.52" fill="#4A4A4A"/>
+<svg class=${styles.svg} xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 295 225">
```
Comment (rijkvanzanten) [inline app/components/parts/author/index.js]: Indentation

File: app/components/parts/type/index.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ export const modalSettings = {
 };
 
 export default () => html`
-<svg class=${styles.svg} xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="302" height="225" viewBox="0 0 302 225">
+<svg class=${styles.svg} xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 302 225">
```
Comment (rijkvanzanten) [inline app/components/parts/type/index.js]: indentation

File: app/index.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ function setupState(state, emitter) {
   Object.assign(state, {
     machineslider: {
       active: false,
-      items: ['genre', 'author', 'type', 'place', 'pages', 'color'],
+      items: ['genre', 'autheur', 'type', 'place', 'pages', 'color'],
```
Comment (rijkvanzanten) [inline app/index.js]: Autheur? 😂

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Some small edits, also; please fix the merge conflicts

File: app/components/parts/author/index.js (hunk 1 above)
Comment (pierman1) [reply app/components/parts/author/index.js]: 👍

File: app/components/parts/author/index.js (hunk 2 above)
Comment (pierman1) [reply app/components/parts/author/index.js]: 👍

File: app/components/parts/type/index.js (hunk 3 above)
Comment (pierman1) [reply app/components/parts/type/index.js]: 👍

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**